### DISCUSSION TITLE
UI feature request: goal - keep finger centred across alpha/number/clipboard modes

### DISCUSSION THREAD

**Discussion by Treeoctopus** [8VIM/8VIM#167] [Ideas]
Hi master-coder & github contributers 
!
8vim just gets better and better! Great job! and open-source, thank goodness!

I have a user interface suggestion that I wanted to run past you. The main alphabet typing is of course focused on the central circle, but the [number] and [clipboard] modes have the red "return to main alpha circle" buttons off to the side.

Whenever anyone switches to one of these modes; whether accidental or on purpose, they're inevitably going to have to press the "return" button again . If it was somehow closer to the center, and in the same location for both of these two modes,  then the ergonomics would be much better.

At present I have to stretch a thumb out to find the "return" button, which feels like a spinning ice skater wobbling out of balance!

Perhaps this could be somehow facilitated by keeping the sidebar persistent

**Comment by flide** [comment]
This is already done.
